src/api/projects/entities/project.entity.ts

```diff
@@ -52,6 +52,9 @@ export class Project extends BaseEntity {
 
   @Field(() => Organization, { nullable: true })
   organization: Organization;
+
+  @Field(() => String, { nullable: true })
```

Comment:
If we place `{ nullable: true }` then the field must have the `?` like `slug?:`. Anyways, this field is required.